apps/admin-be/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "admin-be",
+  "version": "1.0.0",
+  "description": "backend for admin panel",
+  "main": "dist/index.js",
+  "license": "MIT",
+  "scripts": {
+    "devStart": "nodemon --ignore tests/ --watch src -e ts,tsx --exec ts-node src/index.ts",
```

Comment:
lets rename this script to `dev`, so it aligns with the turbo pipeline names (see `turbo.json`).